net/libfab/libfab_internal.h

```diff
@@ -39,16 +40,41 @@ extern struct m0_net_xprt m0_net_libfab_xprt;
  */
 
 struct m0_fab__dom_param {
-	struct fi_info    *fab_fi;      /* Fabric info */
-	struct fi_info    *fab_hints;   /* Fabric info to configure flags */
-	struct fid_fabric *fab_fabric;  /* Fabric fid */
-	struct fid_domain *fab_domain;  /* Domain fid */
+	struct fi_info    *fdp_fi;      /* Fabric info */
+	struct fi_info    *fdp_hints;   /* Fabric info to configure flags */
+	struct fid_fabric *fdp_fabric;  /* Fabric fid */
+	struct fid_domain *fdp_domain;  /* Domain fid */
 };
 
 struct libfab_params {
 	struct fid_mr     *mr; 	/* Memory region to be registered */
 };
 
+struct m0_fab__ep_name {
+	char fen_addr[INET6_ADDRSTRLEN + 6];     /* + 6 for ":<max port no>" */
+};
+
+struct m0_fab__ep_res {
+	struct fi_cq_attr   *fer_cq_attr;   /* Completion Queue attributes */
+	struct fi_av_attr   *fer_av_attr;   /* Address Vector attributes */
+	struct fi_eq_attr   *fer_eq_attr;   /* Event Queue attributes */
+	struct fi_cntr_attr *fer_cntr_attr; /* Counter attributes */
+};
+
+struct m0_fab__ep_param {
```

Comment:
This is the internal representative of a EP?
Then we can rename this to `struct m0_net_fab_ep`?

Reply:
or `struct m0_fab_ep`?